### ISSUE TITLE
3.14.13.0: Doesn't work

### ISSUE THREAD

**Issue by Daniil1288** [uEssentials/uEssentials#239]
Chat doesn't work. AT ALL
pls help

```
[4/15/2016 8:25:32 PM] [Info] [World] Daniil1288 [Daniil1288]: "/god"
[4/15/2016 8:25:32 PM] [Error] InvalidProgramException: Invalid IL code in SDG.Unturned.ChatManager:process (SDG.Unturned.SteamPlayer,string): IL_0019: call      0x0a0004de

 - SDG.Unturned.ChatManager.askChat (CSteamID steamID, Byte mode, System.String text)
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters)
SDG.Unturned.SteamChannel.receive (CSteamID steamID, System.Byte[] packet, Int32 offset, Int32 size)
SDG.Unturned.Provider.receiveServer (CSteamID steamID, System.Byte[] packet, Int32 offset, Int32 size, Int32 channel)
SDG.Unturned.Provider.listenServer (Int32 channel)
SDG.Unturned.Provider.listen ()
SDG.Unturned.Provider.Update ()

```

**Comment by leonardosnt** [comment]
Probably nelson will fix in 3.14.13.1

**Comment by Daniil1288** [comment]
oh no, it's a rocket bug
reported here
https://github.com/RocketMod/Rocket/issues/668
